src/app/components/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import i18n from "i18next";
+// import i18n (needs to be bundled ;))
```

Comment:
I don't understand this comment

Reply:
we use a single JSON file per locale, but when react i18next bundles everything into a single object {en: {} , hi:{}} something like that. so that comment is pointing something like this.

Reply:
Ah ok, do we need that comment though? Does it explain anything the import doesn't?

Reply:
not much but standard documentation keeps this so didn't change that. if you wish I can remove that for sure

Reply:
Ah I see, it's from here:  
https://react.i18next.com/latest/using-with-hooks  
I think it's good to have it in the docs, but it doesn't add any value here.

Reply:
removed that